ynput/OpenPype
# Fusion: Work with pathmaps

## Changelog Description
Path maps are a big part of our Fusion workflow. We map the project folder to a path map within Fusion so all loaders and savers point to the path map variable. This way any computer on any OS can open any comp no matter where the project folder is located.

## Additional info
In OpenPype's default Fusion render location is within the comps root folder. This means every saver node created will get the path `Comp:` and the path will continue from there. This is universal for Fusion and will render on any Fusion renderer.
For any other kind of path map the users have to manually enter the path map in Fusions settings for anything different to happen than before this PR.

This feature would fit perfectly with an updated Fusion Profile functionality where the studio could enter in custom path maps per project and they would be added automatically to Fusions prefs when opened through OP.
This would make the whole system super clean and easy to use.

## Testing notes:
1. Add a path map in Fusions settings that point towards your project folder. Name the path map your project name
2. Create a new saver
3. Check the path and you'll see the start of the path is now `Comp:`
4. Render and publish the footage
5. Load the footage back into the comp
6. Check the path of the loader and you'll see the path now starts with what you named your path map in step 1.
